pkg/controller/podgc/gc_controller.go

```diff
@@ -75,25 +87,31 @@ func (gcc *PodGCController) Run(stop <-chan struct{}) {
 	klog.Infof("Starting GC controller")
 	defer klog.Infof("Shutting down GC controller")
 
-	if !cache.WaitForNamedCacheSync("GC", stop, gcc.podListerSynced) {
+	if !cache.WaitForNamedCacheSync("GC", stop, gcc.podListerSynced, gcc.nodeListerSynced) {
 		return
 	}
 
 	go wait.Until(gcc.gc, gcCheckPeriod, stop)
 
 	<-stop
+	gcc.nodeQueue.ShutDown()
```

Comment:
Move it to defer (right above defer klog.Infof...)

Reply:
Done